dokka-runners/dokka-gradle-plugin/src/main/kotlin/DokkaExtension.kt

```diff
@@ -45,8 +45,29 @@ constructor(
     /** Default Dokka Gradle Plugin cache directory */
     abstract val dokkaCacheDirectory: DirectoryProperty
 
+    /**
+     * The display name used to refer to the module.
+     * It is used for the table of contents, navigation, logging, etc.
+     *
+     * Default: the current project name.
+     */
     abstract val moduleName: Property<String>
+
+    /**
+     * The displayed module version.
+     *
+     * Default: the version of the current project.
+     */
     abstract val moduleVersion: Property<String>
+
+    /**
+     * A distinct relative path for the Dokka Module produced by the current project.
```

Comment:
not sure I understand what this property could be used for when reading KDoc :(

Reply:
I agree, it's unclear. It's difficult to try to explain. I've added a lot more detail.